python/llm/src/bigdl/llm/transformers/qlora.py

```diff
@@ -157,7 +234,7 @@ def _create_new_module(create_new_module_func, lora_config, adapter_name, target
 
 @dataclass
 class LoraConfig(LoraConfigBase):
-
+    lora: bool = field(default=False, metadata={"help": "enable lora"})
     qa_lora: bool = field(default=False, metadata={"help": "enable qa-lora"})
```

Comment:
combine these two into a something like `mode: str =(default="qlora", ...)`

Reply:
Have changed to `training_mode: str` here.